lib/src/accordion/accordion-style.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { DomSanitizer } from "@angular/platform-browser";
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({ name: "dynamicStyle" })
+export class DynamicStylePipe implements PipeTransform {
+    constructor(private sanitizer: DomSanitizer) {}
```

Comment:
where are you using the sanitizer? do you even need it in this file?

Reply:
We are not. It slipped through accidentally.